pandas/tseries/tools.py

```diff
@@ -483,6 +483,49 @@ def parse_time_string(arg, freq=None, dayfirst=None, yearfirst=None):
 normalize_date = tslib.normalize_date
 
 
+# Used in between_time string parsing
+_time_formats = ["%H:%M", "%H%M", "%I:%M%p", "%I%M%p", 
+                 "%H:%M:%S", "%H%M%S", "%I:%M:%S%p", "%I%M%S%p"]
+
+def add_time_format(time_format):
+    _time_formats.append(time_format)
+
+def to_time(arg, format=None):
+    """
+    Parse time strings to time objects using fixed strptime formats.
```

Comment:
this should also accept list-likes for the arg itself (e.g. like ``to_datetime``), ok with it just returning a list if list-like is passed (you can be fancy and return a Series if a Series is passes, ndarray if ndarray, list if list if you want, not much addtl code for that). A scalar will return a scalar. All just like ``to_datetime``